src/common/asb/Asb.c

```diff
@@ -3125,6 +3127,22 @@ static int RemediateEnsureAuditdServiceIsRunning(char* value, OsConfigLogHandle
 {
     int status = 0;
     UNUSED(value);
+
+    // The auoms service is part of Microsoft's Operations Management Suite (OMS) and is used for collecting audit events.
+    // Conflicts between auoms and auditd can arise because both services attempt to manage and collect audit events.
+    // One of the recommended mitigation strategies is Single Service Usage: use either auoms or auditd, but not both.
```

Comment:
Is this our recommendation or a recommendation from somewhere else?

Reply:
Industry recommendation found by our helpful AI companion :)